messages/es.json

```
@@ -3,5 +3,6 @@
   "store/app-store-components-agree-terms": "Al realizar este pedido, acepta los {appTermsAndConditions}",
   "store/app-store-components-terms-and-conditions": " términos y condiciones del {appName}.",
   "store/app-store-components-install-page-button": "Ir a la página de instalación",
-  "store/app-store-components-confirmation-message": "Gracias por su compra!!{newLine}Para completar el proceso de instalación en la cuenta {account}, use el botón de abajo"
+  "store/app-store-components-confirmation-message": "Gracias por su compra!!{newLine}Para completar el proceso de instalación en la cuenta {account}, use el botón de abajo",
+  "store/app-store-components-billing-notice-message": "Se le cobrará una tarifa de $ 0.01 para validar la información de su tarjeta de crédito. Esto será reembolsado tan pronto como se apruebe este pago."
```

A moeda não está atrelada ao idioma da mensagem, provavelmente teria que receber por parâmetro ou emitir (e possivelmente exibir em outro lugar)

```suggestion
  "store/app-store-components-billing-notice-message": "Se le cobrará una tarifa de {currency} 0.01 para validar la información de su tarjeta de crédito. Esto será reembolsado tan pronto como se apruebe este pago."
```